luigi/contrib/ftp.py

```diff
@@ -131,14 +131,11 @@ def _sftp_exists(self, path, mtime):
         return exists
 
     def _ftp_exists(self, path, mtime):
-        path_parts = path.split('/')
-        path = '/'.join(path_parts[:-1])
-        fn = path_parts[-1]
-
-        files = self.conn.nlst(path)
+        dirname = os.path.dirname(path)
```

Comment:
I just looked it up and we can use posixpath.basename (or posixpath.split to get the dirname too) to ensure that this works consistently across systems.